packages/contracts-bedrock/src/L2/GasPriceOracle.sol

```diff
@@ -93,7 +162,23 @@ contract GasPriceOracle is ISemver {
                 total += 16;
             }
         }
-        uint256 unsigned = total + overhead();
+        uint256 unsigned = total + L1Block(Predeploys.L1_BLOCK_ATTRIBUTES).l1FeeOverhead();
         return unsigned + (68 * 16);
     }
+
+    /// @notice Post-ecotone L1 gas estimation calculation.
+    /// @param _data Unsigned fully RLP-encoded transaction to get the L1 gas for.
+    /// @return Amount of L1 gas used to publish the transaction.
+    function _getL1GasUsedEcotone(bytes memory _data) internal pure returns (uint256) {
```

Comment:
this isn't computing a "gas" value, but the approximation of "compressed bytes", so I'd rename it something more accurate to make it better match the spec.

_getCompressedTxSize() ?

Reply:
from Slack discussion: just delete this function after applying the suggested edit above to the "main" function.